applications/rubintv/values-usdfdev.yaml

```diff
@@ -0,0 +1,89 @@
+siteTag: "usdf-dev"
+
+frontend:
+  debug: true
+  env:
+  - name: http_proxy
+    value: "http://sdfproxy.sdf.slac.stanford.edu:3128"
+  - name: https_proxy
+    value: "http://sdfproxy.sdf.slac.stanford.edu:3128"
+  - name: no_proxy
+    value: ".sdf.slac.stanford.edu,.slac.stanford.edu,localhost,127.0.0.1"
+  image:
+    tag: tickets-DM-40149
+    pullPolicy: Always
+
+workers:
+  replicas: 1
+  image:
+    repository: lsstts/rubintv-broadcaster
+    tag: c0030_usdf
+    pullPolicy: Always
+  uid: 15517
+  gid: 4085
+  env:
+  - name: http_proxy
+    value: "http://sdfproxy.sdf.slac.stanford.edu:3128"
+  - name: https_proxy
+    value: "http://sdfproxy.sdf.slac.stanford.edu:3128"
+  - name: squid_proxy
+    value: "http://sdfproxy.sdf.slac.stanford.edu:3128"
+  - name: no_proxy
+    value: ".sdf.slac.stanford.edu,.slac.stanford.edu,localhost,127.0.0.1"
+  - name: HTTP_PROXY
+    value: "http://sdfproxy.sdf.slac.stanford.edu:3128"
+  - name: HTTPS_PROXY
+    value: "http://sdfproxy.sdf.slac.stanford.edu:3128"
+  - name: SQUID_PROXY
+    value: "http://sdfproxy.sdf.slac.stanford.edu:3128"
+  - name: NO_PROXY
+    value: ".sdf.slac.stanford.edu,.slac.stanford.edu,localhost,127.0.0.1"
+  - name: DEPLOY_BRANCH
+    value: "deploy-rubintv-worker"
+  - name: S3_ENDPOINT_URL
+    value: "https://s3dfrgw.slac.stanford.edu"
+  - name: DAF_BUTLER_REPOSITORY_INDEX
+    value: "s3://rubin-summit-users/data-repos.yaml"
+  - name: DAF_BUTLER_REPOSITORY
+    value: "/sdf/group/rubin/repo/ir2/butler.yaml"
+  envSecrets:
+  - name: ASTROMETRY_NET_API_KEY
+    secretName: rubintv-secret
+    secretKey: astrometry-net-api-key
+  imagePullSecrets:
+  - name: pull-secret
```

Comment:
Maybe I'm blind, but I don't see where this is used in the template.

Reply:
Injected.